### PR TITLE
cesc/maxmsp maxobj missing stuff

### PR DESCRIPTION
**Pull request by FrancescElies**
Takes stuff out of #66475 to make review process easier

### CODE REVIEW

File: types/maxmsp/maxmsp-tests.ts [hunk 1]
```diff
@@ -200,6 +201,7 @@ myMaxobj.setattr("openrect", [0, 0, 0, 0]);
 const boxattrnames = myMaxobj.getboxattrnames();
 [r1, r2, r3, r4] = myMaxobj.getboxattr("presentation_rect") as Rect;
 myMaxobj.setboxattr("presentation_rect", [0, 0, 0, 0]);
+myMaxobj.setboxattr("presentation_rect", 0, 0, 0, 0);
```
Comment (FrancescElies) [inline types/maxmsp/maxmsp-tests.ts:204]: boths works
grafik
```js
autowatch = 1;

var myinlet = patcher.newobject("inlet");
var myinlet2 = patcher.newobject("inlet");
myinlet.setboxattr("patching_rect", [40, 40, 400, 400]);
myinlet2.setboxattr("patching_rect", 0, 0, 40, 40);

```

File: types/maxmsp/maxmsp-tests.ts [hunk 2]
```diff
@@ -2,12 +2,16 @@
...
 jsarguments.push(1, 2, 3);
-error('This is a test error.');
-cpost('This is cpost.');
-post('This is a post.');
+error('This is a test error.\n');
+error('This is a', 'multiple string', 'error\n');
+cpost('This is cpost.\n');
+cpost('This is a', 'multiple string', 'cpost\n');
+post('This is a post.\n');
+post('This is a', 'multiple string', 'post\n');
```
Comment (FrancescElies) [inline types/maxmsp/maxmsp-tests.ts:14]: grafik

```js
autowatch = 1
error('This is a test error.\n');
error('This is a', 'multiple string', 'error\n');
cpost('This is cpost.\n');
cpost('This is a', 'multiple string', 'cpost\n');
post('This is a post.\n');
post('This is a', 'multiple string', 'post\n');
```

File: types/maxmsp/index.d.ts [hunk 3]
```diff
@@ -1211,6 +1212,15 @@ declare class Maxobj {
...
     js: any;
 
+    /**
+     * Whether patchcords are connected to the object's inlets and outlets and, if so,
+     * the connected objects. Returns a generic object with two arrays,
+     * 'inputs' and 'outputs', of MaxobjConnection objects.
+     * These have the properties 'srcobject', 'dstobject', 'srcoutlet' and 'dstinlet'
+     * which can be used to walk the graph from JS.
+     */
+    patchcords: { inputs: MaxobjConnection[]; outputs: MaxobjConnection[] };
```
Comment (FrancescElies) [inline types/maxmsp/index.d.ts:1222]: see patchcords example
grafik

**Review by twhiston — APPROVED** [review]
Thanks for submitting these changes!

**Comment by FrancescElies** [comment]
Ready to merge

**Pull request merged**